lib/import/admin.js

```diff
@@ -129,6 +136,7 @@ function save_features(feature_collection) {
     feat.name = feat.properties.name;
     feat.admin_code = feat.properties.admin_code;
     feat.geo_area_sqkm = feat.properties.geo_area_sqkm;
+    feat.population = feat.properties.population;
```

Comment:
population field is set to:
    type: Number,
    required: false,

With this configuration, string values are ignored and don't face validation testing. Will leave the validation in anyway.